modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/UnsafeHTML.js

```
@@ -48,7 +48,7 @@ export default class UnsafeHTML extends React.PureComponent {
 
 		const scriptElements = Array.from(
 			this.state.ref.querySelectorAll('script')
-		);
+		).filter((script) => script.type !== 'text/x-alloy-template');
```

I think we should check
```js
!script.type || script.type === 'text/javascript'
```
just to be sure we are only executing js tags.

Moreover I think that if there is an error inside the script, an error is thrown here:
https://github.com/p2kmgcl/liferay-portal/pull/641/files#diff-338f0733688bfc3bc30a201e8f6b2eefR62

I need to verify this, but if that's true, ¿should we capture the error or let it propagate?

---

Makes total sense yeah, I'll update that

Regarding the error, I think there isn't a case where we can do anything about this error 🤔 So propagating i think is fine